showcase/src/assets/translations/fr.json

```diff
@@ -346,6 +346,14 @@
           "TITLE": "Basé sur Angular Material"
         },
         "TITLE": "Typography"
+      },
+      "LAYOUT": {
+        "TITLE": "Layout",
+        "INTRO": "Nous vous suggérons d'utiliser <a href=\"https://github.com/angular/flex-layout#angular-flex-layout\" target=\"_blank\" rel=\"noopener noreferrer\">Angular Flex-Layout</a> comme aide lors de la construction de la présentation de votre application. Une documentation complète est disponible sur leur <a href=\"https://github.com/angular/flex-layout/wiki/API-Documentation\"  target=\"_blank\" rel=\"noopener noreferrer\">wiki</a>.",
```

Comment:
Nous vous suggérons d'utiliser ... pour vous aider lors de la construction de l'interface(?) de votre application. 

Not sure about the "interface" thingy, it's just that "présentation" seems a bit odd in french.